surveys/sphinx-survey-2025.yml

```diff
@@ -155,7 +155,7 @@ questions:
   - "I don't know"
 
 # Motivation: Understand the constellation of tools for documentation this responder works with.
-- title: "What markup language are you using in your documentation?"
+- title: "What markup language do you like using in your documentation?"
```

Comment:
I think we care more about what people want rather than what they use. e.g. they may use reStructuredText because it's built in but would rather have Markdown.

These are two seperate topics and we could also ask both. But if limited to one of them, what they want feels more important.

Reply:
I suppose changing this question relates to why we're asking it. What does preferred/actual markup language tell us? Respondents may have a different preference from what they actually use, but inertia, network effects, business decisions, etc mean that they don't change.

Reply:
Exactly, and what can we do with that information? How will this guide our future development decisions?

- I think the actually used language alone doesn't tell us much. I anticipate there are a lot of reST users because the survey is from a sphinx context and reST is the default language. - We basically could even scrape GitHub to get current language usage. - But what actions are influenced by that insight?

- answer on what is actually used.- For the language preference, the implications are clearer. If people are satisfied with reStructuredText, we don't have specific actions. If they rather like markdown, better integration with MyST may be an option.

Reply:
These could actually be two distinct questions, but we originally opted to ask about the markup they use right now to map the current usage.
The "what markup you like" is too vague and does not necessarily mean that someone would choose/change to something they like/enjoy using/are more familiar with.

Reply:
```suggestion
- title: "What markup language are you using in your documentation?"
```